examples/README.md

```diff
@@ -17,19 +17,19 @@ By default, [`configuration.nix`](configuration.nix) enables `bitcoind` and `cli
   This is the fastest way to set up a node.\
   Requires: [Nix](https://nixos.org/), a systemd-based Linux distro and root privileges
 
+- [`./deploy-container-minimal.sh`](deploy-container-minimal.sh) creates a
+  container defined by [standalone-configuration.nix](standalone-configuration.nix).\
+  You can copy and import this file to use nix-bitcoin in an existing NixOS configuration.\
+  The configuration doesn't use the [secure-node.nix](../modules/presets/secure-node.nix) preset.\
+  Requires: [Nix](https://nixos.org/), a systemd-based Linux distro and root privileges
```

Comment:
I think that if I were skimming paragraph, I might read the first sentence, think it only applied if you want to run `nix-bitcoin` in a container, and miss the rest.

So maybe it could be broken out into its own section:

```suggestion

-  You can copy and import this [standalone-configuration.nix](standalone-configuration.nix) to use nix-bitcoin in an existing NixOS configuration.\
  The configuration doesn't use the [secure-node.nix](../modules/presets/secure-node.nix) preset.\
  Requires: [Nix](https://nixos.org/), a systemd-based Linux distro and root privileges
```